plugins/placeholdertext/plugin.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @fileOverview Simple CKEditor 4 plugin that adds placeholder text to the editor.
+ */
+( function() {
+	/**
+	 * Namespace providing configuration for placeholdertext plugin..
+	 *
+	 * @singleton
+	 * @class CKEDITOR.plugins.placeholdertext
+	 * @member CKEDITOR.plugins
+	 */
+	var pluginNamespace = {
+		/**
+		 * Template, which will be used to create placeholder element.
+		 *
+		 * `{PLACEHOLDER}` is replaced by the value of CKEDITOR.config.placeholdertext.
+		 *
+		 * @property {String}
+		 */
+		template: '<p data-cke-placeholdertext>{PLACEHOLDER}</p>',
```

Comment:
We already have API for templates https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_template.html

Reply:
I've removed the template altogether.